### PR TITLE
#168252582 Gets All Users Articles

### PR DESCRIPTION
**Pull request by Adekoreday**
#### What does this PR do?
- Enables user to get all their articles

#### Description of Task proposed in this pull request?
- article controller
- article swagger docs
- article test

#### How should this be manually tested (Quality Assurance)?
```
login
create articles
GET localhost:9000/api/v1/articles/getUserArticles 
```

#### What are the relevant pivotal tracker stories?
- [168252582](https://www.pivotaltracker.com/story/show/168252582)

#### Any background context you want to add (Operations Impact)?
- N/A

#### What I have learned working on this feature: 
- I understood how to use sequelize findAllAndCount method

### CODE REVIEW

File: server/routes/article.js [hunk 1]
```diff
@@ -46,4 +46,11 @@ router.patch(
   articleEditValidation,
   Articles.update
 );
+
+router.get(
+  '/userArticles',
+  verifyToken,
+  getSessionFromToken,
+  Articles.userArticles
```
Comment (chialuka) [inline server/routes/article.js:55]: Why are we not checking whether the user is verified?

Comment (Adekoreday) [reply server/routes/article.js:55]: fixed thanks

**Review by JuwonAbiola — DISMISSED** [review]

**Review by ayodejiAA — DISMISSED** [review]

**Review by JuwonAbiola — DISMISSED** [review]

File: test/articles/getUserArticles.test.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  context(
+    'when the get User Articles method encounters a internal error',
+    () => {
+      it('throws a server error', async () => {
+        const userArticles = sinon
+          .stub(Article, 'findAndCountAll')
+          .callsFake(() => {});
+        const response = await chai
+          .request(app)
+          .get(`${BASE_URL}/articles/userArticles`)
```
Comment (henryade) [inline test/articles/getUserArticles.test.js]: Please kindly change this route name. I dont believe a route should have a camel case string

**Review by henryade — DISMISSED** [review]

File: test/articles/getUserArticles.test.js (hunk 2 above)
Comment (Adekoreday) [reply test/articles/getUserArticles.test.js]: I have done that thanks

**Review by dinobi — DISMISSED** [review]

**Review by chialuka — APPROVED** [review]

**Review by JuwonAbiola — APPROVED** [review]

**Review by dinobi — APPROVED** [review]

**Pull request merged**